paths.h

```diff
@@ -87,6 +87,7 @@ void pv_paths_storage_trail_pvr_file(char *buf, size_t size, const char *rev,
 
 #define UNCLAIMED_FNAME "unclaimed.config"
 #define PANTAHUB_FNAME "pantahub.config"
+#define MOUNTED_FNAME "mounted"
```

Comment:
lets name it .pv_mounted .... same we probably would want to add to all disks we mount (e.g. crypt disks)... and later then also gather space of those disks of course...

Reply:
done